cmd/config.go

```diff
@@ -46,13 +56,42 @@ func (config *ConfigDir) Get() string {
 	return config.dir
 }
 
+func (config *ConfigDir) getCertsDir() string {
+	return filepath.Join(config.Get(), certsDir)
+}
+
+// GetCADir - returns certificate CA directory.
+func (config *ConfigDir) GetCADir() string {
+	return filepath.Join(config.getCertsDir(), certsCADir)
+}
+
+// Create - creates configuration directory tree.
+func (config *ConfigDir) Create() error {
+	return mkdirAll(config.GetCADir(), 0700)
+}
+
+// GetMinioConfigFile - returns absolute path of config.json file.
+func (config *ConfigDir) GetMinioConfigFile() string {
```

Comment:
I think GetMinioConfigPath() is more expressive than GetMinioConfigFile(), the same thing for GetPublicCertFile() and GetPrivateKeyFile()